base/abstractarray.jl

```diff
@@ -325,15 +325,15 @@ end
 checkbounds(A::AbstractArray) = checkbounds(A, 1) # 0-d case
 
 """
-    checkbounds_indices(Bool, IA, I)
+    checkbounds_indices(Bool, A, IA, I)
```

Comment:
Ah, yup, DArray extends it as an optimization. It'll be easy to add the new methods right alongside it.